vllm/config.py

```diff
@@ -2697,12 +2720,19 @@ def verify_lora_support(self):
                 "V1 LoRA does not support long LoRA, please use V0.")
 
 
+@config
 @dataclass
 class PromptAdapterConfig:
-    max_prompt_adapters: int
-    max_prompt_adapter_token: int
+    max_prompt_adapters: int = 1
+    """Max number of PromptAdapters in a batch."""
+    max_prompt_adapter_token: int = 0
+    """Max number of PromptAdapters tokens."""
     max_cpu_prompt_adapters: Optional[int] = None
-    prompt_adapter_dtype: Optional[torch.dtype] = None
+    """Maximum number of PromptAdapters to store in CPU memory. Must be >= than
+    `max_prompt_adapters`."""
+    prompt_adapter_dtype: Union[torch.dtype, str] = "auto"
```

Comment:
```suggestion
    prompt_adapter_dtype: Optional[torch.dtype] = None
```

Looks like a copy and paste error

Reply:
The logic is already in `PromptAdapterConfig.verify_with_model_config`. Also, since prompt adapters are planned for deprecation I didn't really want to go the extra mile on this feature.